packages/web/jest.setup.js

```diff
@@ -1,5 +1,5 @@
 // add some helpful assertions
-import 'jest-dom/extend-expect'
+//import 'jest-dom/extend-expect';
```

Comment:
Doing this for now, since it was not being used.

A start to fix it if we want to use `jest-dom/extend-expect`
https://github.com/gnapse/jest-dom/issues/45#issuecomment-408149403

Reply:
no problem, open an issue so we can improve this later on